### ISSUE TITLE
Accessibility: fails when Label for an input field is hidden

### ISSUE THREAD

**Issue by adventmedia** [formio/formio.js#6268]
**Describe the bug**
Input components with Hide Label (Text Field, Text Area etc.) fail accessibility tests as the related <label> element is not rendered. 

**Version/Branch**
@fomio/js: 5.2.2
@formio/react: 6.1.0

**To Reproduce**

1. Create form
2. Add input components — any that have "Hide Label" option
3. View form and run accessibility evaluation (tested with Evince.com)

Accessibility scan flags these components as missing "Accessible name"

Suggested solution: if the "Hide Label" option is enabled, apply the label text to an "aria-label" attribute of the input component.

Similar problems observed with Radio, Checkbox and Select input components

**Comment by lane-formio** [comment]
After review, we don't expect to resource this, but are open to reviewing PRs. Otherwise would recommend unhiding the label.
